src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -182,13 +182,13 @@ public string FindSymbolFilePath(string pdbFileName, Guid pdbIndexGuid, int pdbI
             }
 
             string pdbIndexPath = null;
-            string pdbSimpleName = Path.GetFileName(pdbFileName);        // Make sure the simple name is really a simple name
+            string pdbSimpleName = PathUtil.GetPlatformIndependentFileName(pdbFileName);        // Make sure the simple name is really a simple name
 
             // If we have a dllPath, look right beside it, or in a directory symbols.pri\retail\dll
             if (pdbPath == null && dllFilePath != null)        // Check next to the file. 
             {
                 m_log.WriteLine("FindSymbolFilePath: Checking relative to DLL path {0}", dllFilePath);
-                string pdbPathCandidate = Path.Combine(Path.GetDirectoryName(dllFilePath), Path.GetFileName(pdbFileName));
+                string pdbPathCandidate = Path.Combine(Path.GetDirectoryName(dllFilePath), PathUtil.GetPlatformIndependentFileName(pdbFileName));
```

Comment:
Just doing a quick search for `Path.GetFileName`, there are many more of these.  I support wanting to run more workloads using TraceEvent on Linux.  At the same time, I'd like to understand a bit more about what you're looking to do.  We probably also need to start thinking more broadly about how the TraceEvent codebase is going to deal with these sorts of platform-specific things.  The pattern you've set forth here might be the right one, but I've not thought about it much.